components/hmpo-character-count/macro.njk

```diff
@@ -0,0 +1,7 @@
+{% macro hmpoCharacterCount(ctx, params, base) %}
```

Comment:
We'd need to add these new fields to [hmpo-field](https://github.com/HMPO/hmpo-components/blob/master/components/hmpo-field/macro.njk) to allow them to be auto-rendered on `form-template` pages